pkgs/top-level/python-packages.nix

```diff
@@ -6338,6 +6338,31 @@ in {
   pyprof2calltree = callPackage ../development/python-modules/pyprof2calltree { };
 
   hcloud = callPackage ../development/python-modules/hcloud { };
+
+  stups = callPackage ../development/python-modules/stups-berry { };
+
+  stups-berry = callPackage ../development/python-modules/stups-berry { };
+
+  stups-cli-support = callPackage ../development/python-modules/stups-cli-support { };
+
+  stups-fullstop = callPackage ../development/python-modules/stups-fullstop { };
+
+  stups-kio = callPackage ../development/python-modules/stups-kio { };
+
+  stups-pierone = callPackage ../development/python-modules/stups-pierone { };
+
+  stups-piu = callPackage ../development/python-modules/stups-piu { };
+
+  stups-senza = callPackage ../development/python-modules/stups-senza { };
+
+  stups-tokens = callPackage ../development/python-modules/stups-tokens { };
+
+  stups-zign = callPackage ../development/python-modules/stups-zign { };
+
+  zalando-kubectl = callPackage ../development/python-modules/zalando-kubectl { };
+
+  zalando-aws-cli = callPackage ../development/python-modules/zalando-aws-cli { };
+
```

Comment:
please make an attempt to drop this alphabetically amongst the other packages :)